### ISSUE TITLE
SublimeMerge updated yesterday and broke my license

### ISSUE THREAD

**Issue by huornlmj** [sublimehq/sublime_merge#1902]
**Version info**

 - OS: Windows 10
 - Build: 2096

**Description**

I've been using SublimeMerge for years with a paid license.  Yesterday I got a notification of an update available which I accepted.  Now it says it's unlicensed.  Tried to put my license back in again and it says the license is expired!  This should not have happened.  It should only have allowed for updates within the version that my license covers, and not force a new version that's outside my license scope!!!

How do I go back to the version that my license covers?

**Comment by jfcherng** [comment]
> How do I go back to the version that my license covers?

You can download arbitrary version of ST/SM by change the version number in its download link.
https://download.sublimetext.com/sublime_merge_build_2096_x64_setup.exe

**Comment by huornlmj** [comment]
> > How do I go back to the version that my license covers?
> 
> You can download arbitrary version of ST/SM by change the version number in its download link. https://download.sublimetext.com/sublime_merge_build_2096_x64_setup.exe

Thanks, but the issue is that I don't know what version I had before the upgrade.  Had I known it was going to dump me out of license I would have not only recorded the version but just not have updated.

**Comment by jfcherng** [comment]
> > > How do I go back to the version that my license covers?
> > 
> > 
> > You can download arbitrary version of ST/SM by change the version number in its download link. [download.sublimetext.com/sublime_merge_build_2096_x64_setup.exe](https://download.sublimetext.com/sublime_merge_build_2096_x64_setup.exe)
> 
> Thanks, but the issue is that I don't know what version I had before the upgrade. Had I known it was going to dump me out of license I would have not only recorded the version but just not have updated.

`Menu > Help > About Sublime Merge` should show the day you purchased it. License is valid for 3-year update. Release date can be found on changelog: https://www.sublimemerge.com/dev

**Comment by huornlmj** [comment]
> `Menu > Help > About Sublime Merge` should show the day you purchased it. License is valid for 3-year update. Release date can be found on changelog: https://www.sublimemerge.com/dev

It just says `Unregistered` now.

All I want is to go back to the version I had before this new update announced itself and reapply my license.

**Comment by jfcherng** [comment]
> It just says `Unregistered` now.

How about let's register?

**Comment by Zander-Labuschagne** [comment]
I'm also having this issue, installing an older version within the license period still says it is unregistered. But I did not do an update, the one I had installed just started to block any mouse clicks with no other popup in the way, I could not even close it and had to use Windows Task Manager. I then looked for a different version and came across this issue. Older versions do not block my mouse input though.

**Comment by jfcherng** [comment]
> I'm also having this issue, installing an older version within the license period still says it is unregistered.

Then just do register...

It shouldn't be `unregistered` after you register it anyway. E.g., mine has been expired

![image](https://github.com/sublimehq/sublime_merge/assets/6594915/cf9ac072-fc7e-4617-9391-241fccab7833)

**Comment by huornlmj** [comment]
Mine was registered too, until the update the other day.  I would not have taken any update if it said it was going to knock my license out of use!

**Comment by jfcherng** [comment]
> Mine was registered too, until the update the other day. I would not have taken any update if it said it was going to knock my license out of use!

To be clear, people (including me) can't help that. What I try to help is to have you rollback to the version you want to use. But I see no more question about this now.

**Comment by huornlmj** [comment]
Understood.  So would you know what the version was before the 2096?  If I can get that then I might be able to find the link to that version setup file and downgrade and then enter my registration code.

**Comment by jfcherng** [comment]
> Understood. So would you know what the version was before the 2096? If I can get that then I might be able to find the link to that version setup file and downgrade and then enter my registration code.

I think I clearly answered that in https://github.com/sublimehq/sublime_merge/issues/1902#issuecomment-2074991184

**Comment by huornlmj** [comment]
Unfortunately it doesn't tell me which version was available before 2096.  It's a link to 2096.  I don't want to assume the version I was using before it offered an update was 2095 or even it 2095 exists for example.

**Comment by jfcherng** [comment]
> Unfortunately it doesn't tell me which version was available before 2096. It's a link to 2096. I don't want to assume the version I was using before it offered an update was 2095 or even it 2095 exists for example.

glad I have a big screen and my mouse scrolls.

![image](https://github.com/sublimehq/sublime_merge/assets/6594915/1acdaf6f-0aa0-4d3e-a7e4-931ea692844f)

**Comment by Zander-Labuschagne** [comment]
> I'm also having this issue, installing an older version within the license period still says it is unregistered. But I did not do an update, the one I had installed just started to block any mouse clicks with no other popup in the way, I could not even close it and had to use Windows Task Manager. I then looked for a different version and came across this issue. Older versions do not block my mouse input though.

My mistake, the version I was using here was still outside the license term, using an appropriate version solved the issue.

**Comment by huornlmj** [comment]
Downgrading to 2095 didn't work either.

**Comment by jfcherng** [comment]
> Downgrading to 2095 didn't work either.

Please define "didn't work".

**Comment by huornlmj** [comment]
I've contacted Sublime support.  Seems to be the best option.

**Comment by jfcherng** [comment]
Unless you have a precise description, or give a screenshot (which may speak more thing), I don't think there will be a difference to ask another person.

**Comment by dpjohnst** [comment]
Hi @huornlmj,

Thanks for reaching out and sorry this was your experience. This is definitely not the intended experience when updating.
If your license isn't covered by an update, the update dialog is meant to indicate this.

I'm going to investigate why this wasn't the case, and get back to you.

Could you confirm if downgrading to 2091 (the previous stable build) resolves the issue for you:
https://download.sublimetext.com/sublime_merge_build_2091_x64_setup.exe

Kind regards,
\- Dylan from Sublime HQ

**Comment by dpjohnst** [comment]
G'day @huornlmj,

Thanks for reaching out (and for using Sublime Merge) - our support team has contacted you via email.

Many thanks,
\- Dylan from Sublime HQ

**Comment by huornlmj** [comment]
@dpjohnst I've followed your suggestion and downgraded to 2091 and it now says it's registered again.  However, even after I unchecked the "Check for updates periodically" box, it still went out and checked for updates and I got this again this morning, and it looks the exact same as it did the last time.  No warning that my license is expired or accepting this update will void my warranty again...
![image](https://github.com/sublimehq/sublime_merge/assets/28444739/ff87c931-4815-4499-af4d-30812e335f74)

**Comment by dpjohnst** [comment]
Hi @huornlmj,

Just following up on this - our license server is in the process of being upgraded, so previous builds may temporarily work with expired subscriptions. This will change once the license server has completed the upgrade.

There is also a bug in 2091 where the "Check for updates periodically" feature requires a restart of Sublime Merge to take effect.

Hope this clarifies things.

Kind regards,
\- Dylan from Sublime HQ

**Comment by huornlmj** [comment]
Thanks @jfcherng .  But as I have an apparently expired license I cannot move to 2091, so I will need to continue ignoring the prompt.

**Comment by jfcherng** [comment]
"Check for updates periodically" works even if your license expires. But the staff said there is a bug about it in SM 2091 so 🤷 .

**Comment by huornlmj** [comment]
Yes, but Sorry, I am on 2091 (getting the versions mixed up).  So I confirmed that I have this bug where it checks for updates even if I deselect the box to do this.
